src/instructlab/configuration.py

```
@@ -361,6 +361,12 @@ class _evaluate(BaseModel):
     gpus: Optional[int] = Field(
         default=None, description="Number of GPUs to use for running evaluation."
     )
+    results: Optional[str] = Field(
```

Would format be a better term?

---

Originally I actually did use `format` but I got a MyPy error about overriding built-ins

---

It might be worth it to squash the error.  Unless there is another commonly used variation other than format.

---

Will do :+1: